ch05.adoc

```diff
@@ -183,13 +187,16 @@ _This section has been superseded by the treatment of time series as a type of d
 [[grid-mappings-and-projections, Section 5.6, "Horizontal Coordinate Reference Systems, Grid Mappings, and Projections"]]
 === Horizontal Coordinate Reference Systems, Grid Mappings, and Projections
 
-When the coordinate variables for a horizontal grid are not longitude and latitude, it is required
-that the true latitude and longitude coordinates be supplied via the **`coordinates`** attribute.  A
-__grid mapping variable__ is required if, in addition, it is desired to describe the mapping between the
-given coordinate variables and the true latitude and longitude coordinates, or to describe the
+A __grid mapping variable__ may be referenced by a data variable: to describe the mapping between the
```

Comment:
I think this sentence doesn't represent the intent clearly. How about this language for lines 190-195?

A __grid mapping variable__ may be referenced by a data variable in order to explicitly declare the coordinate reference system (CRS) used for the spatial coordinate values. For example, if the spatial coordinates are latitude and longitude, the __grid mapping variable__ can be used to declare the figure of the earth (WGS84 ellipsoid, sphere, etc.) they are based on. If the spatial coordinates are easting, northing, and elevation in a map projection, the __grid mapping variable__ declares the map projection CRS used and provides the information needed to calculate latitude and longitude from easting and northing. If the spatial coordinates are not georeferenced coordinates, the __grid mapping variable__ can make it clear that this is the case.

If the spatial coordinates are not longitude and latitude and no __grid mapping variable__ is provided, then it is recommended that auxiliary latitude and longitude coordinates be included in the file and referenced via the **`coordinates`** attribute. These auxiliary coordinate variables provide the latitude and longitude for each element of the data variable.

Reply:
Hello @JimBiardCics 

> If the spatial coordinates are not georeferenced coordinates, the **grid mapping variable** can make it clear that this is the case.

I'm wary of this statement.  I'm not sure that it is possible to make it clear that this is the case.

> If the spatial coordinates are not longitude and latitude and no grid mapping variable is provided, then it is recommended that auxiliary latitude and longitude coordinates be included in the file and referenced via the coordinates attribute. 

This is a content change, not just a rewording.  In the current standard and this proposed change, this is mandated, not recommended, I believe.  Is it your intent to change this mandate to a recommends, or did you intend this to remain a mandate?

thank you
marqh

Reply:
@marqh The non-georeferenced case is described by @hrajagers in [an earlier comment to this PR](https://github.com/cf-convention/cf-conventions/pull/133#issuecomment-410691784).

Regarding the content change you noted: I'm OK with making it mandatory to include lat and lon if you haven't otherwise declared the CRS for a pair of X and Y coordinate variables.

Reply:
thank you for the feedback @JimBiardCics 

I agree that the multi-phrase sentence at line 190 is convoluted; I have adopted you suggested text for this.
I don't yet agree that the non-georeferenced case is high profile and should be used as an example.  Indeed the earlier comment relies on WKT-CRS, which is at the moment an extension to grid_mappins, and for experts, I suggest.

I have kept the previous text on the conditional mandate of either a grid_mapping, or explicit latitude and longitude, as I think it is clearer.

your continued feedback on this text is very much welcome
marqh

Reply:
@marqh The non-georeferenced case can wait until someone specifically requests it, but it is a valid use case that was mentioned in the threads on this change. Let's leave it out if it is going to slow down progress on this change.